### PR TITLE
Fixes issue #799 remove todos from pywbemtools/pywbemtools and subdirectories

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Removes all TODOs from /pywbemtools/pywbemtools directory and subdirectories.

See commit for details about each removal.

Note that this change involved:

1. moving a couple to ISSUEs. 2 new issues.
2. Removing some as worthless
3. Cleaning up some documentation
4. One code change.

NOTE: This fixes issue#799 despite fact that branch defines 773

### CODE REVIEW

File: pywbemtools/pywbemcli/_common.py [hunk 1]
```diff
@@ -1599,7 +1593,7 @@ def all_classnames_depsorted(namespace, conn):
         dep_classnames = dependent_classnames(cls)
         all_deps[cls.classname] = set(dep_classnames)
 
-    # TODO: Add support for case insensitive sorting.
+    # ISSUE $ 954: Add support for case insensitive sorting.
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_common.py:1596]: In other places, we use the notation `#NNN`, and I suggest to be consistent eg for searching.

File: pywbemtools/pywbemcli/_display_cimobjects.py [hunk 2]
```diff
@@ -490,8 +490,8 @@ def _display_instances_as_table(insts, table_width, table_format,
     else:
         max_cell_width = table_width
 
-    # TODO Future: Decide whether and how the showing of units should be
-    #              controlled.
+    # ISSYE #953 Future: Decide whether and how the showing of units should be
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_display_cimobjects.py:493]: Typo "ISSYE"

**Review by andy-maier — CHANGES_REQUESTED** [review]
Two minor comments

File: pywbemtools/pywbemcli/_display_cimobjects.py (hunk 2 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_display_cimobjects.py:493]: Fixed.

File: pywbemtools/pywbemcli/_common.py (hunk 1 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_common.py:1596]: Fixed

**Review by andy-maier — APPROVED** [review]

**Pull request merged**